cgp/genome.py

```diff
@@ -346,6 +349,27 @@ def _is_active_input_gene(self, gene_idx: int) -> bool:
         else:
             assert False  # should never be reached
 
+    def _select_gene_indices_for_mutation(self, mutation_rate, len_dna, rng):
+        """Selects the gene indices for mutations
+
+        Parameters
+        ----------
+        mutation_rate : float
+            Proportion of genes to be mutated, between 0 and 1.
```

Comment:
```suggestion
            Probability of a gene to be mutated, between 0 (excluded) and 1 (included).
```
btw, you should also check in other doc strings whether the description of `mutation_rate` needs to be updated.